Straight_skeleton_2/test/Straight_skeleton_2/issue7284.cpp

```diff
@@ -0,0 +1,220 @@
+  "1894527831184417839378056821257055353472302650834937455815530384810719018309831565538833463663255005630695677027768372926535599763538463931956509965375033109667984089527593199501872609943009121207/758336587013273798592720716302823769966064331932243246386637484212455144968828064928657276042528413567014683523373308742479021698836129960571049486997667032336755041184105565416394283737415680000",
+  "30389989540782110473728368050411374004243546950498015672047447812803467121256072555535515206724150663447848525329584897663494658714072084450000107051376363228978910165903680034560249671271183078564398465823808862174829626602313047918535338173571189199866854086024041290216043222967075370554943273699854284994451825104618058549658575885359038795601485461944836824766538133922906091528657345111794924752045906921638768210632413666299415149614999326139594859450382083547541673128448932409041691827446243108750640934605945368256919813424762222040778893253981881370638388841247676488965131625307567265935828645861603063324797349538508067585211513158790642061476896878599852740888495853896189177403777086512656320645349274359329088652547797849/11073014429281006666193246290917387908002648863865545740375717690997389816571506685325962528448018484963488232749909444009593245542959973206435082342862459479567309550167455283077556604198292338385221037274304994552092354502415456710081329301517274939747123788424811655302721178400051177604747816784274204431647691260045616979957814284225160433591757285799829577512273609614408785038703926056405556424111457207787968525989421207941935465591309552308497720454208919141604812685542888672898769666762812377512922493164653250964938093742264282225459034506065267458773463636370375185635172304136716462183278384946600916823611103113472346728977026142767670973846112122509041041946537691280004353652224139965392928513762839244316143026962432000"
+};
+
+#ifndef CGAL_SLS_TEST_USE_EPECK_WITH_SQRT
+I_Polygon_2 exact_to_inexact_poly(const E_Polygon_2& poly)
+{
+  E_to_I converter;
+
+  std::vector<I_Point_2> boundary;
+  std::transform(poly.vertices_begin(), poly.vertices_end(),
+                 std::back_inserter(boundary),
+                 [&](const E_Point_2& p) { return converter(p); });
+
+  return I_Polygon_2(boundary.begin(), boundary.end());
+}
+#endif
+
+void test(const std::vector<std::string>& x,
+          const std::vector<std::string>& y)
+{
+#ifdef CGAL_SLS_TEST_USE_EPECK_WITH_SQRT
+  std::vector<I_Point_2> poly_vec;
+  for (size_t i = 0; i < x.size(); ++i)
+    poly_vec.emplace_back(I_FT(x[i]), I_FT(y[i]));
+
+  I_Polygon_2 ipoly (poly_vec.begin(), poly_vec.end());
+#else
+  std::vector<E_Point_2> poly_vec;
+  for (size_t i = 0; i < x.size(); ++i)
+    poly_vec.emplace_back(E_FT(mpq_class(x[i])), E_FT(mpq_class(y[i])));
```

Comment:
Instead of `mpq_class`   we should try `E_FT::ET`   That is an undocumented typedef and is the exact type used by `Lazy_exact_nt`. @sloriot can you give it a try.